frontend/packages/schema-editor/src/components/SchemaInspector/restrictions/ArrayRestrictions.test.tsx

```diff
@@ -1,40 +1,79 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render as rtlRender, screen, waitFor } from '@testing-library/react';
+import type { RestrictionItemProps } from '../ItemRestrictions';
 import { ArrayRestrictions } from './ArrayRestrictions';
 import { ArrRestrictionKeys } from '@altinn/schema-model';
-import { mockUseTranslation } from '../../../../../../testing/mocks/i18nMock';
+import userEvent from '@testing-library/user-event';
+import { textMock } from '../../../../../../testing/mocks/i18nMock';
 
 // Test data:
-const restrictionKeyTexts: { [key in ArrRestrictionKeys]: string } = {
-  [ArrRestrictionKeys.minItems]: 'Minimum antall elementer',
-  [ArrRestrictionKeys.maxItems]: 'Maksimum antall elementer',
-  [ArrRestrictionKeys.uniqueItems]: 'Unike elementer',
+const onChangeRestrictionValueMock = jest.fn();
+const pathMock = '#/properties/xxsfds';
+
+const render = (props: Partial<RestrictionItemProps> = {}) => {
+  const allProps = {
+    path: pathMock,
+    readonly: false,
+    restrictions: [],
+    onChangeRestrictions: jest.fn(),
+    onChangeRestrictionValue: onChangeRestrictionValueMock,
+    ...props,
+  } as RestrictionItemProps;
+  return rtlRender(<ArrayRestrictions {...allProps} />);
```

Comment:
Vi vil unngå type-casting hvis det ikke er nødvendig.
```suggestion
  const allProps: RestrictionItemProps = {
    path: pathMock,
    readonly: false,
    restrictions: [],
    onChangeRestrictions: jest.fn(),
    onChangeRestrictionValue: onChangeRestrictionValueMock,
    ...props,
  };
  return rtlRender(<ArrayRestrictions {...allProps} />);
```